### ISSUE TITLE
SSL Method Development and Training.

### ISSUE THREAD

**Issue by MohammedSB** [MohammedSB/RadiologyFM#4]
A big, somewhat open question now is whether DINOv2 pre-training works for radiological images. We will use DINOv2 pretrained weights as a training initialization, but we don't need to use DINOv2 pre-training, the SSL method we will use depends on what works best. 

I have received messages that DINOv2 could possibly be worse that DINO for pathology (potentially because of the masking adopting from IBOT), but the evidence is contradictory, and it is not on radiology. This work for example showcases that it works well for pathology https://arxiv.org/pdf/2309.07778.pdf. But they don't provide a comparison with other SSL methods.

This issue is for people who want to try out using DINOv2 pre-training on radiology images, and investigating ways of adapting it for radiology if it does not work out of the box.
